src/components/features/PostForm/PostForm.js

```diff
@@ -57,15 +64,28 @@ const PostForm = ({action, actionText, ...props}) => {
           <DatePicker selected={publishedDate} onChange={(date) => setPublishedDate(date)}/>
           {dateError && <small className="d-block form-text text-danger mt-2">Please input the date</small>}
         </Form.Group>
+        <Form.Group>
+          <Form.Label>Category</Form.Label>
+          <Form.Select 
+            value={category} 
+            aria-label="category" 
+            {...register("category", {required: true})}
+            onChange={e => setCategory(e.target.value)}>
+            <option value=''>Open this select menu</option>
```

Comment:
Tego nie powinno dać się wybrać
https://stackoverflow.com/questions/5805059/how-do-i-make-a-placeholder-for-a-select-box